src/test_runner.cr

```diff
@@ -269,8 +270,9 @@ module Mint
       terminal.puts "  #{ARROW} #{@failed.size} failed"
 
       @failed.each do |failure|
-        terminal.puts "    #{failure.name}".colorize(:red)
-        terminal.puts "    |> #{failure.result}".colorize(:red)
+        terminal.puts "    #{failure.suite}".colorize(:red)
+        terminal.puts "    - #{failure.name}".colorize(:red)
+        terminal.puts "      |> #{failure.result}".colorize(:red)
```

Comment:
I'd group it failures by the suit name so we won't clutter the output with duplicates.